server/util/clickhouse/clickhouse.go

```diff
@@ -259,6 +255,112 @@ func (e *Execution) AdditionalFields() []string {
 	}
 }
 
+// TestTargetStatus represents the status of a target, the target info and invocation details
+type TestTargetStatus struct {
```

Comment:
out of curiosity, why `TestTargetStatus` instead of `TargetStatus` like the mysql table name?

Reply:
This table only includes test targets. I assume that we will have different schemas for build targets.